packages/lesswrong/components/common/ForumIcon.tsx

```diff
@@ -5,6 +5,8 @@ import SpeakerWaveIcon from "@heroicons/react/24/solid/SpeakerWaveIcon";
 import BookmarkIcon from "@heroicons/react/24/solid/BookmarkIcon";
 import StarIcon from "@heroicons/react/24/solid/StarIcon";
 import UserIcon from "@heroicons/react/24/solid/UserIcon";
+import BellOutlineIcon from "@heroicons/react/24/outline/BellIcon";
+import BellIcon from "@heroicons/react/24/solid/BellIcon";
```

Comment:
Ah nice, I have also added this in one of my PRs

Reply:
Oh which one?